build.gradle

```diff
@@ -8,8 +8,10 @@ buildscript {
 
     ext {
         opensearch_group = "org.opensearch"
-        opensearch_version = "${opensearch_version}"
-        common_utils_version = "${opensearchBaseVersion}.0-SNAPSHOT"
+        opensearch_version = System.getProperty("opensearch.version", "1.3.0-SNAPSHOT")
+        // 1.3.0 -> 1.3.0.0, and 1.3.0-SNAPSHOT -> 1.3.0.0-SNAPSHOT
```

Comment:
Is 1.x.0 -> 1.x.0.0 a convention that we normally use for how the version evolve?

Reply:
Yes, other plugins like AD using same convention.